src/components/HistoricalWeatherDataTab/HistoricalWeatherDataTab.tsx

```diff
@@ -1,62 +1,65 @@
 import { useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
-import { useSnackbar } from 'notistack';
-import CircularProgress from '@mui/material/CircularProgress';
 import pick from 'lodash/pick';
+import { useSnackbar } from 'notistack';
 import HistoricalWeatherForm from 'components/HistoricalWeatherForm';
 import HistoricalWeatherWidget from 'components/HistoricalWeatherWidget';
-import ForecastBody from 'types/forecast';
+import Spinner from 'components/Spinner';
 import WeatherSearchCaption from 'components/WeatherSearchCaption';
+import ForecastBody from 'types/forecast';
+import sleep from 'utils/sleep';
 import { FormValuesType } from '../HistoricalWeatherForm/validation';
-import FORECAST from './forecast';
+import FIXTURE from './fixture';
 import * as classes from './styles';
 
-const SPINNER_SIZE = 25;
-
-export default function HistoricalForecast() {
+export default function HistoricalWeatherDataTab() {
   const { enqueueSnackbar } = useSnackbar();
-  const [selectedSearchParams, onSelectSearchParams] =
+  const [searchParams, handleSubmitSearchParams] =
     useState<FormValuesType | null>(null);
   const [forecast, setForecast] = useState<ForecastBody[] | null>(null);
   const [isLoading, setIsLoading] = useState<boolean>(false);
   const { t } = useTranslation();
+  const DELAY = 1000;
```

Comment:
Move me out of the component. Such constants in SCREAMING_SNAKE_CASE shouldn't be in component's body